lib/Server.js

```diff
@@ -147,6 +148,10 @@ class Server {
   }
 
   setupProgressPlugin() {
+    new webpack.ProgressPlugin({
+      profile: this.profile,
+    }).apply(this.compiler);
```

Comment:
hm, we really need two plugins? I think we can use one instance of plugin

Reply:
I think 2 are needed since one supplies a custom handler but the other uses the default handler: https://github.com/webpack/webpack/blob/master/lib/ProgressPlugin.js#L103

Reply:
hm, we need investigate this and tests this, looks like we setup two progress plugin and it can be invalid usage

Reply:
@evilebottnawi Wouldn't it be the equivalent of two plugins tapping into the exact same events? Assuming the plugin was made properly.

Reply:
@evilebottnawi Are my changes here causing the Windows CI problems? I'm unsure why progress plugin outputs stuff to `stderr`, but maybe that is causing a problem? https://github.com/webpack/webpack/blob/master/lib/ProgressPlugin.js#L49